include/openenclave/debugmalloc.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_DEBUG_MALLOC_H
+#define _OE_DEBUG_MALLOC_H
+
+#include <openenclave/bits/result.h>
+
+#ifdef __cplusplus
+extern "C"
+{
+#endif
+
+    oe_result_t oe_debug_malloc_tracking_start(void);
```

Comment:
Since this is published API, it needs to have doxygen documentation comments.

Reply:
Updated, thanks!